libs/components/data-manager/src/assets/locales/resources_en_US.json

```diff
@@ -11,18 +11,34 @@
     "_description": "Text for the clear button in the data manager column picker",
     "message": "Cancel"
   },
+  "skyux_data_manager_select_all_button_aria_label": {
+    "_description": "Default aria-label text for the select all button in the data manager multiselect toolbar",
```

Comment:
Do we need "Default" in these descriptions? I notice that the corresponding `button_title` descriptions don't start with "Default." Starting with "Default" implies that the value is editable, so if user can edit these aria-label values, then I would keep it. However, if they can't edit the values, then I'd delete "Default" and capitalize "Aria-label" at the start. (Either that or edit the `button_title` properties to start with "Default" for consistency, but I think deleting "Default" in the aria-label descriptions is the better option.)

Reply:
Done